### PR TITLE
Add xengsort

### PR DESCRIPTION
**Pull request by JZentgraf**
Adding xengsort to bioconda.
Added a recipe using the pypi package and adapting the meta.yaml.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by JZentgraf** [comment]
@BiocondaBot please add label

File: recipes/xengsort/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ed168b6c073eedec670efce7e2dda775039e3b38703dab9f58f494367391a511
+
+build:
+  number: 0
+  entry_points:
+    - xengsort = xengsort.xengsort.xengsort_main:main
+  noarch: generic
```
Comment (mencian) [inline recipes/xengsort/meta.yaml:16]:
```suggestion
  noarch: python
```

File: recipes/xengsort/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  imports:
+    - xengsort
+  commands:
+    - xengsort --help
+    - xengsort index --help
+    - xengsort classify --help
+
+about:
+  home: https://gitlab.com/genomeinformatics/xengsort
+  license: MIT
```
Comment (mencian) [inline recipes/xengsort/meta.yaml:40]:
```suggestion
  license: MIT
  license_file: LICENSE
```

**Review by mencian — CHANGES_REQUESTED** [review]
A couple of comments in-line, thanks.

**Comment by JZentgraf** [comment]
Thanks for the comments!
I accepted both.

**Review by mencian — APPROVED** [review]

**Pull request merged**